CHANGELOG.md

```diff
@@ -5,7 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/cockroachdb/cockroach-operator/compare/v2.2.0-beta.2...master)
+# [Unreleased](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...master)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
+
+[2.2.0](https://github.com/cockroachdb/cockroach-operator/compare/2.2.0...2.2.0)
```

Comment:
Mm...this doesn't seem right. None of these 2.2.0...2.2.0 ones should be here. Did the script run multiple times or something?

Reply:
Oh, yeah, I had a bunch of false starts with the script for various reasons. I'll delete all the `2.2.0...2.2.0` ones